src/hypothesis/internal/compat.py

```diff
@@ -290,16 +290,8 @@ def inner(*args, **kwargs):
         getfullargspec = silence_warnings(getfullargspec)
 
 
-def get_type_hints(thing):
-    """Try to return any type hints for ``thing``."""
-    try:
-        import typing
-        return typing.get_type_hints(thing)
```

Comment:
It had some workarounds for `get_type_hints` sometimes failing on 3.5 too didn't it?